cmd/stat_test.go

```diff
@@ -18,19 +18,22 @@
 package cmd
 
 import (
-	"os"
-	"reflect"
-	"strings"
-	"testing"
-	"time"
+    "os"
```

Comment:
Your local indentions are wrong, run gofmt to ensure the formatting is correct and also use goimports to fix the import.

Reply:
I ran gofmt on stat_test.go